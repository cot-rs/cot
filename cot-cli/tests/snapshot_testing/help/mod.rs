use super::*;

#[test]
fn no_args() {
    assert_cmd_snapshot!(cot_cli!());
}

#[test]
fn short_help() {
    assert_cmd_snapshot!(cot_cli!("-h"));
}

#[test]
fn long_help() {
    assert_cmd_snapshot!(cot_cli!("--help"));
}

#[test]
fn version() {
    assert_cmd_snapshot!(cot_cli!("--version"));
}

#[test]
fn help() {
    assert_cmd_snapshot!(cot_cli!("help"));
}

#[test]
fn help_new() {
    assert_cmd_snapshot!(cot_cli!("help", "new"));
}

#[test]
fn help_migration() {
    assert_cmd_snapshot!(cot_cli!("help", "migration"));
}

#[test]
fn help_migration_list() {
    assert_cmd_snapshot!(cot_cli!("help", "migration", "list"));
}

#[test]
fn help_migration_make() {
    assert_cmd_snapshot!(cot_cli!("help", "migration", "make"));
}

#[test]
fn help_cli_manpages() {
    assert_cmd_snapshot!(cot_cli!("help", "cli", "manpages"));
}

#[test]
fn help_cli_completions() {
    assert_cmd_snapshot!(cot_cli!("help", "cli", "completions"));
}
