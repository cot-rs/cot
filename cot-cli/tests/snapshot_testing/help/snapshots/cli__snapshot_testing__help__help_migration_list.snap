---
source: cot-cli/tests/snapshot_testing/help/mod.rs
info:
  program: cot
  args:
    - help
    - migration
    - list
---
success: true
exit_code: 0
----- stdout -----
List all migrations for a Cot project

Usage: cot migration list [OPTIONS] [PATH]

Arguments:
  [PATH]  Path to the crate directory to list migrations for  [default: current directory]

Options:
  -v, --verbose...  Increase logging verbosity
  -q, --quiet...    Decrease logging verbosity
  -h, --help        Print help

----- stderr -----
